Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -584,3 +584,77 @@ extension AlertManager: PresetActivationObserver {
         }
     }
 }
+
+// MARK: - Issue/Retract Alert Permissions Warning
+extension AlertManager: AlertPermissionsCheckerDelegate {
+    func alertPermissions(requiresRiskMitigation: Bool, scheduledDeliveryEnabled: Bool) {
+        if !issueOrRetract(alert: AlertPermissionsChecker.unsafeNotificationPermissionsAlert,
+                           condition: requiresRiskMitigation,
+                           alreadyIssued: UserDefaults.standard.hasIssuedRiskMitigatingAlert,
+                           setAlreadyIssued: { UserDefaults.standard.hasIssuedRiskMitigatingAlert = $0 },
+                           issueHandler: { alert in
+            // the risk mitigation in-app alert is presented with a button to navigate to settings
+            self.recordIssued(alert: alert)
+            let alertController = AlertPermissionsChecker.constructUnsafeNotificationPermissionsInAppAlert() { [weak self] in
+                self?.acknowledgeAlert(identifier: AlertPermissionsChecker.unsafeNotificationPermissionsAlertIdentifier)
```

Comment:
We really should be adding actions to our Alert model for this. I'm guessing you're not doing that for expediency in this case.